src/main/java/htsjdk/tribble/Feature.java

```diff
@@ -27,13 +27,14 @@
 import htsjdk.samtools.util.Locatable;
 
 /**
- * Represents a locus on a reference sequence.   All Features are expected to return 1-based closed-ended intervals.
+ * Marker interface for locatables with Tribble support.
```

Comment:
Might make things clearer to add an extra sentence here: "Marker interface for Locatables with Tribble support. A Feature represents a record in a tribble-supported file format."